assets/javascripts/discourse/mixins/sidebars.js.es6

```diff
@@ -231,7 +263,12 @@ export default Mixin.create({
     if (hasRightSidebar) {
       offset += mainRightOffset;
     }
+    if (hasLeftSidebar && leftFull) {
+      offset = 0;
+      root.style.setProperty('overflow-x', 'hidden');
```

Comment:
@keegangeorge Two things on this. Firstly, you shouldn't use a computed property callback (i.e. this function) to perform another "set" operation. If you need to set something else as a result of a computed property changing, use a seperate funciton. Always keep computed property functions focused on single type of output, in this case the styles being applied to the main element. 

If you need to perform other behaviour as a result of changes to the same properties observed by ``mainStyle`` set up a seperate computed property or observer to do that.